src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+  --min-box-height: 300px;
+  --max-row-width: 1199px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  max-width: var(--max-row-width);
```

Comment:
```suggestion
```

Let's not use max-width for this task and use 'flex-basis' instead. You result looks not like it should be:

![1](https://user-images.githubusercontent.com/66570142/113029233-d9313480-9194-11eb-85d9-ec19abaa0653.png)

Two rows should be stretched to full width of the page